config/manager/manager.yaml

```diff
@@ -10,6 +10,8 @@ spec:
     matchLabels:
       control-plane: controller-manager
   replicas: 1
+  strategy:
+    type: Recreate
```

Comment:
Recreate is ok for the Operator. It means there will downtime on updates, but that's will only affect the Operator itself, not Grafana.